server/test/run_repository.go

```diff
@@ -602,7 +603,7 @@ func (r *runRepository) GetTransactionRunSteps(ctx context.Context, id id.ID, ru
 WHERE transaction_run_steps.transaction_run_id = $1 AND transaction_run_steps.transaction_run_transaction_id = $2
 ORDER BY test_runs.completed_at ASC
 `
-	query, params := sqlutil.Tenant(ctx, query, runID, id)
+	query, params := sqlutil.Tenant(ctx, query, strconv.Itoa(runID), id)
```

Comment:
this is required for the new pgx db  driver

Reply:
Question: what is the idea of using `pgx` instead of `pq` directly?

Reply:
pgx has support for `LISTEN` / `NOTIFY`. Originally this PR was going to do everything but it got larger than expected so I'll implement that in a separated PR, but we already did the driver change at the beginning. The idea was to check that everything worked with the new driver